sdk/storage/azure-storage-blob/azure/storage/blob/_shared/base_client.py

```diff
@@ -378,7 +379,7 @@ def create_configuration(**kwargs):
     # type: (**Any) -> Configuration
     config = Configuration(**kwargs)
     config.headers_policy = StorageHeadersPolicy(**kwargs)
-    config.user_agent_policy = StorageUserAgentPolicy(**kwargs)
+    config.user_agent_policy = UserAgentPolicy(base_user_agent=USER_AGENT,**kwargs)
```

Comment:
Please use "sdk_moniker" instead. That would simplify to something like 
```python
"storage-blob/{}".format(VERSION)
```
https://github.com/Azure/azure-sdk-for-python/blob/ea88267383a965dbd015a29d82f233d3391b19c2/sdk/core/azure-core/azure/core/pipeline/policies/_universal.py#L198

Reply:
That will give us the version of azure-core instead of storage